package.json

```diff
@@ -79,11 +81,14 @@
     "@types/semver": "^7.5.0",
     "@types/tmp": "^0.2.3",
     "@types/yargs": "^17.0.32",
+    "@typescript-eslint/core": "workspace:^",
+    "@typescript-eslint/eslint-plugin-internal": "workspace:^",
     "console-fail-test": "^0.2.3",
+    "cross-env": "^7.0.3",
```

Comment:
```suggestion
```
Looks like `cross-env` is unused?


```plaintext
$ rg cross-env

packages/repo-tools/package.json
21:    "cross-env": "*",

docs/linting/Troubleshooting.mdx
359:- Running with [`--trace-deprecation`](https://nodejs.org/api/cli.html#--trace-deprecation) (e.g. `npx cross-env NODE_OPTIONS=--trace-deprecation npm run lint`)

package.json
87:    "cross-env": "^7.0.3",

yarn.lock
5353:    cross-env: "*"
5501:    cross-env: ^7.0.3
7826:"cross-env@npm:*, cross-env@npm:^7.0.3":
7828:  resolution: "cross-env@npm:7.0.3"
7832:    cross-env: src/bin/cross-env.js
7833:    cross-env-shell: src/bin/cross-env-shell.js
```

Reply:
I had intended to use it for script(s) to enforce things use flat config - but it turns out that setting `ESLINT_USE_FLAT_CONFIG=true` isn't actually required - ESLint will actually just default to using the `eslint.config.js` if one exists and if the env var isn't set.